records_mover/records/prep_and_load.py

```diff
@@ -15,6 +15,7 @@ def prep_and_load(tbl: TargetTableDetails,
                   load: Callable[[DBDriver], Optional[int]],
                   reset_before_reload: Callable[[], None] = lambda: None) -> MoveResult:
     logger.info(f"Connecting to database...")
+    # TODO: Can this method maybe take a loader rather than a driver?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker